.bazelrc.common

```diff
@@ -21,9 +21,16 @@ startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
 #          and developer mode is enabled.
 build --enable_runfiles
 
-# Turn off legacy external runfiles
-# This prevents accidentally depending on this feature, which Bazel will remove.
-build --nolegacy_external_runfiles
+common --enable_platform_specific_config
+
+# Turn off legacy external runfiles on all platforms except Windows.
+# This prevents accidentally dependening on this feature, which Bazel will remove.
```

Comment:
nit: spelling

(why is my brain like that?)

Reply:
:scream_cat: I even carefully proofread this..